test/e2e/import_test.go

```diff
@@ -20,29 +20,27 @@ limitations under the License.
 package e2e
 
 import (
-	"fmt"
 	"path/filepath"
 	"strconv"
 
 	"github.com/drone/envsubst/v2"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"github.com/rancher-sandbox/rancher-turtles/internal/rancher"
+	provisioningv1 "github.com/rancher-sandbox/rancher-turtles/internal/rancher/provisioning/v1"
 	"sigs.k8s.io/cluster-api/controllers/remote"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	. "sigs.k8s.io/controller-runtime/pkg/envtest/komega"
 
+	turtelesnaming "github.com/rancher-sandbox/rancher-turtles/util/naming"
```

Comment:
same nit, can we please check for it repo-wide and fix at once: 
```suggestion
	turtlesnaming "github.com/rancher-sandbox/rancher-turtles/util/naming"
```